shanoir-ng-datasets/src/main/java/org/shanoir/ng/dataset/service/DatasetDownloaderServiceImpl.java

```diff
@@ -229,16 +229,30 @@ private void reconvertToNifti(String format, HttpServletResponse response, Long
 		List<String> files = new ArrayList<>();
 		for (File res : workFolder.listFiles()) {
 
+			String datasetFilePath = res.getAbsolutePath();
+			String fileName = res.getName();
+			boolean compressed = datasetFilePath.endsWith(".nii");
```

Comment:
Maybe this variable should be call "toCompress" (or "toZip") ?

Reply:
Done